recipes-multimedia/alsa/alsa-utils/soundconfig

```diff
@@ -0,0 +1,187 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+# setup alsa (especially the mixer config)
+
+mixer() {
+  parm=${4:-on}
+  amixer -c "$1" sset "$2" "$3" $parm >/dev/null 2>&1
+  amixer -c "$1" sset "$2" $parm >/dev/null 2>&1
+}
+
+(
+. /etc/profile
+
+dtsoc() {
+    # /proc/device-tree/compatible contains null characters that acts like a separators
+    # hence usage of strings
+    cat /proc/device-tree/compatible | strings | cut -f1,2 -d',' | tail -n 1
+}
```

Comment:
That's dtsoc - it doesn't work without strings inside the pipeline. I suppose it is due to busybox or something.